src/Util/RequestAttributesExtractor.php

```diff
@@ -56,6 +56,12 @@ public static function extractAttributes(Request $request)
             throw new RuntimeException('One of the request attribute "_api_collection_operation_name" or "_api_item_operation_name" must be defined.');
         }
 
+        if (null === $apiRequest = $request->attributes->get('_api_request')) {
+            $result['request'] = true;
+        } else {
+            $result['request'] = (bool) $apiRequest;
```

Comment:
Yes me too but the cast is mandatory to support the routing component < 3.2 (before this version, types wasn't supported and the user was able to put anything that can be casted to a boolean).